src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -354,6 +356,8 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Import Project="Microsoft.NET.Sdk.Razor.MvcApplicationPartsDiscovery.targets" Condition="'$(_TargetingNETCoreApp30OrLater)' == 'true'" />
 
+  <Import Project="Microsoft.NET.Sdk.Razor.Components.Wasm.targets" Condition="'$(_TargetingNET50OrLater)' == 'true' AND '$(UseBlazorWebAssembly)' == 'true'" />
```

Comment:
This target gets imported far too late to set up useful defaults such as `RuntimeIdentifier` or the flag that tells the SDK not to append the rid to the output path. We are able to do other things such as tell it to not reference M.AspNetCore.App or do trimmed publish. One way to remedy the evaluation order issue is to add a blazor-wasm specific SDK.